packages/graphback-runtime/src/GraphbackPage.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Interface represents GraphbackPage type used for findAll queries
+ */
+export interface GraphbackPage {
```

Comment:
This is already defined: https://github.com/aerogear/graphback/blob/master/packages/graphback-runtime/src/data/GraphbackDataProvider.ts#L9-L12

Reply:
I decided to extract it out and keep it separate from GraphbackDataProvider as I am using Page type in a number of places